### ISSUE TITLE
Qupzilla 2: Can't play mp4 video in html5 video tag

### ISSUE THREAD

**Issue by mushroomsir** [QupZilla/qupzilla#1973]
Do you plan to support it in the future version?
`Qt WebEngine supports the MPEG-4 Part 14 (MP4) file format only if the required proprietary audio and video codecs`
http://doc.qt.io/qt-5/qtwebengine-platform-notes.html

**Comment by cranes-bill** [comment]
It depends of your repository, if you are using precompiled version of QtWebEngine, otherwise it's completely up to you.

**Comment by mushroomsir** [comment]
@cranes-bill  I found qt 5.7 is support for DRM-protected HTML5 video, therefore, we don't  need to build own QtWebEngine. Just wait for qt 5.7 coming.
https://wiki.qt.io/New_Features_in_Qt_5.7

**Comment by cranes-bill** [comment]
I'm already using my own build of QtWebEngine. I wrote the above, just to explain that playing of `MP4 files` is possible. Anyway, thanks for your attention! :thumbsup:

**Comment by mathieujobin** [comment]
QupZilla is really amazing ! So lightweight

thank for you making this browser

**Comment by fcore117** [comment]
as i understand qt 5.7 includes h264 play out of box without those nasty OS dependant codecs?

**Comment by nowrep** [comment]
@fcore117 No, Qt 5.7 only adds support for DRM restricted content, but you still need to rebuild QtWebEngine in order to enable proprietary codecs.

**Comment by fcore117** [comment]
Please build some official version and upload to non US servers, ignore those DMCA monkeys. Lot of people would be grateful for this. chromium.woolyss.com builds with h264 for example.

**Comment by keshavbhatt** [comment]
@cranes-bill i need those qt libs bro i downloaded and build qt57's webengine module and it failed two times , it takes lot of time so i need those libs , i need them in my project please mail me keshavnrj@gmail.com

**Comment by cranes-bill** [comment]
Just append `WEBENGINE_CONFIG+="use_proprietary_codecs"` while executing **qmake**.

**Comment by keshavbhatt** [comment]
@cranes-bill am getting errors :( can you please share libs .

**Comment by cranes-bill** [comment]
You want precompiled libs, right?
If true, please note that above i was talking about Qt version 5.6.1. Also perhaps you are using different system, so i'm not sure how you can get it to works...
That's it. 
If you still want it, just tell me and i will sand you what you want!
Sorry for my bad English!

**Comment by keshavbhatt** [comment]
@cranes-bill am good with 5.6.1 libs with codecs support man please upload the somewhere so i can get them.

**Comment by keshavbhatt** [comment]
@cranes-bill if src was compiled with  WEBENGINE_CONFIG+=use_proprietary_codecs
then i want them very urgently please upload them i will be thankful to you bro .

**Comment by cranes-bill** [comment]
To avoid spam here, please check your mail!

**Comment by Killerbeans** [comment]
Hi @cranes-bill and @keshavbhatt, please I hope you all doing great. I have the same issue with compiling webengine and plus my system is slow. Can you please email me the precompiled qt webengine 5.7 with codecs enabled. My email: bibsilvers@gmail.com. Thanks a lot.
